packages/grid/_modules_/grid/hooks/root/useEvents.ts

```diff
@@ -117,6 +111,9 @@ export function useEvents(gridRootRef: React.RefObject<HTMLDivElement>, apiRef:
   useGridApiOptionHandler(apiRef, GRID_ROW_ENTER, options.onRowEnter);
   useGridApiOptionHandler(apiRef, GRID_ROW_LEAVE, options.onRowLeave);
 
+  useGridApiOptionHandler(apiRef, GRID_RESIZE, options.onResize);
```

Comment:
There're still calls to `useGridApiOptionHandler` outside of `useEvents`. Shouldn't all of them be moved here? If `option.onSomething` is called after the default behavior, user won't be able to cancel it. Related to https://github.com/mui-org/material-ui-x/issues/1403#issuecomment-818777963.

Reply:
No because `useGridApiOptionHandler` places the listener in the first position of the listeners array so the handler can stop the propagation of the event.